index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+module.exports = {
+  extends: [
+    "plugin:@typescript-eslint/recommended"
+  ],
+  rules: {
+    "@typescript-eslint/explicit-function-return-type": 0,
+    "@typescript-eslint/no-unused-vars": 0,
+    "@typescript-eslint/explicit-member-accessibility": 0,
+    "@typescript-eslint/no-explicit-any": 0,
+    "@typescript-eslint/camelcase": 0,
+    "@typescript-eslint/ban-ts-ignore": 0,
+    "@typescript-eslint/no-non-null-assertion": 0,
+    "@typescript-eslint/explicit-module-boundary-types": 0,
+    "@typescript-eslint/ban-ts-comment": 0
+  }
```

Comment:
We need to enable `no-shadow` from `@typescript-eslint` otherwise we may encounter issues with TypeScript only features such as enums. Likewise, we need to enable `no-use-before-define` from `@typescript-eslint` as well to avoid inconsistencies. Note that when doing these we need to disable corresponding base rules that come from core ESLint.